src/openms/include/OpenMS/FORMAT/HANDLERS/XQuestResultXMLHandler.h

```diff
@@ -63,10 +63,10 @@ namespace OpenMS
 
       /**
        * @brief Removes a substring from a larger string
-       * @param large String from which `small` should be removed
-       * @param small Substring to be removed from `large`
+       * @param large String from which `tiny` should be removed
+       * @param tiny Substring to be removed from `large`
        */
-      static void removeSubstring(String & large, const String & small);
+      static void removeSubstring(String & large, const String & tiny);
```

Comment:
would it make sense to move this into OpenMS::String() ?
And maybe add a while loop or document that only the first occurence is removed.